bindings/rust/bench/src/harness.rs

```diff
@@ -118,28 +128,26 @@ macro_rules! test_tls_bench_harnesses {
     $(
         mod $lib_name {
             use super::*;
+            use CipherSuite::*;
```

Comment:
I use the * imports because I need to iterate through all of the possible values of the enum and I don't want to write out `ECGroup::` every time, especially when we need to loop through more enums as features get added.